README.md

```diff
@@ -28,8 +28,7 @@ What does it do?
 | CPython 3.7     | ✅ |   | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
 | CPython 3.8     | ✅ |   | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
 | CPython 3.9     | ✅ | ✅ | ✅  | ✅  | ✅ | ✅ | ✅ | ✅ | ✅ |
-| PyPy 3.6 v7.3.3 | ✅ |    |    | ✅  | ✅ |    |    |    |   |
-| PyPy 3.7 (beta) v7.3.3 | ✅ | |  | ✅  | ✅ |    |    |   |    |
+| PyPy 3.7 v7.3.4 | ✅ |   | ✅  |    | ✅ |    |    |    |   |
```

Comment:
```suggestion
| PyPy 3.7 v7.3 | ✅ |   | ✅  |    | ✅ |    |    |    |   |
```

IMHO, the patch version is not needed because the ABI tag only relates to PyPy being 3.7 v7.3.*.
That's one less thing to remember to update and it's already what's being done for CPython.

Reply:
Good idea, I hadn't fully twigged this. The Python/ABI tag is `pp36-pypy36_pp73`. We don't list patch version of CPythons, so let's remove the PyPy patch version it from here.

Reply:
Nice, thanks! Done.